accepted/color-4-new-spaces.md

```diff
@@ -1141,26 +1141,17 @@ The procedure is:
     * Let `split-last` be the result calling `string.split()` with the last
       element of `input` as the string to split, and `/` as the separator.
 
-    * If `split-last` has two items, and one or both items are an unquoted
-      string that's case-insensitively equal to 'none':
+    * If `split-last` doesn't have exactly two items, throw an error.
 
-      > Special handling for `none/none`, `none/<number>`, and `<number>/none`.
+    > This solves for a legacy handling of `/` in Sass that would produce an
```

Comment:
Should this note be moved to in between lines 1139 and 1141? It seems like this entire section is meant for that rather than just the bullet points below 🤔